cypress/e2e/inventory/call-number/browse-call-numbers-result-list-displays-preceding-result.cy.js

```diff
@@ -0,0 +1,116 @@
+import BrowseCallNumber from '../../../support/fragments/inventory/search/browseCallNumber';
+
+describe('inventory', () => {
+  describe('Call Number Browse', () => {
+    let callNumberIncrement = 30;
+    let barcodeId = 111;
+    const numberOfInventory = 25;
+    const callNumbers = [];
+    const barcodes = [];
+    [...Array(numberOfInventory)].forEach(() => barcodes.push(`399095${++barcodeId}`));
+    [...Array(numberOfInventory)].forEach(() => callNumbers.push(`E 3184 S75 12${++callNumberIncrement}`));
+    const testData = {
+      userServicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+    };
+
+    before('Create test data', () => {
+      cy.getAdminToken().then(() => {
+        ServicePoints.createViaApi(testData.userServicePoint);
+        testData.defaultLocation = Location.getDefaultLocation(testData.userServicePoint.id);
+        Location.createViaApi(testData.defaultLocation);
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((holdingTypes) => {
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: getTestEntityValue('type'),
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 })
+          .then((materialTypes) => {
+            testData.materialTypeId = materialTypes.id;
+            testData.materialType = materialTypes.name;
+          })
+          .then(() => {
+            barcodes.forEach((barcode, i) => {
+              InventoryInstances.createFolioInstanceViaApi({
+                instance: {
+                  instanceTypeId: testData.instanceTypeId,
+                  title: `instance_${randomFourDigitNumber()}`,
+                },
+                holdings: [
+                  {
+                    holdingsTypeId: testData.holdingTypeId,
+                    permanentLocationId: testData.defaultLocation.id,
+                    callNumber: callNumbers[i],
+                  },
+                ],
+                items: [
+                  {
+                    barcode: barcodes[i],
+                    status: { name: ITEM_STATUS_NAMES.AVAILABLE },
+                    permanentLoanType: { id: testData.loanTypeId },
+                    materialType: { id: testData.materialTypeId },
+                  },
+                ],
+              });
+            });
+          });
+      });
+      cy.createTempUser([Permissions.inventoryAll.gui]).then((userProperties) => {
+        testData.user = userProperties;
+
+        cy.login(testData.user.username, testData.user.password, {
+          path: TopMenu.inventoryPath,
+          waiter: InventoryInstances.waitContentLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
```

Comment:
please also delete loan type like this `cy.deleteLoanType(testData.loanTypeId);`

Reply:
done